test/test_cpp_extensions_stream_and_event.py

```diff
@@ -81,21 +81,30 @@ def setUpClass(cls):
     def test_stream_event(self):
         s = torch.Stream()
         self.assertTrue(s.device_type, int(torch._C._autograd.DeviceType.MTIA))
-        e = torch.Event()
+        e = torch.Event(enable_timing=True)
+        e1 = torch.Event(enable_timing=True)
+        e1.record()
         self.assertTrue(e.device.type, "mtia")
         # Should be nullptr by default
         self.assertTrue(e.event_id == 0)
         s.record_event(event=e)
         print(f"recorded event 1: {e}")
         self.assertTrue(e.event_id != 0)
+        # The enable_timing of event created by record_event() is false
         e2 = s.record_event()
         print(f"recorded event 2: {e2}")
         self.assertTrue(e2.event_id != 0)
         self.assertTrue(e2.event_id != e.event_id)
         e.synchronize()
+        e1.synchronize()
         e2.synchronize()
-        time_elapsed = e.elapsed_time(e2)
-        print(f"time elapsed between e1 and e2: {time_elapsed}")
```

Comment:
Hey, sorry to bother @albanD again, please help review the latest commit.

I am not familiar with MTIA, because enable_timing of e2 is false, so in theory `time_elapsed = e.elapsed_time(e2)` should fail, but it does not.

Reply:
Yeah the fake implementation for testing in https://github.com/pytorch/pytorch/blob/459484d74029fb3e0872fa5402167832eca17415/test/cpp_extensions/mtia_extension.cpp#L128 doesn't do that check.

I think it's ok to skip the test here and @egienvalue can change that in a follow up if you want the test impl to properly behave here.

Reply:
Got it, thank you.